docs/guides/installation.md

```diff
@@ -108,6 +108,12 @@ Export your GitHub personal access token as an environment variable:
 export GITHUB_TOKEN=<your-token>
 ```
 
+!!! hint "Deploy Key"
+    The bootstrap command creates a ssh key which it stores as a secret in the
+    Kubernetes cluster. The key is also used to create a deploy key in the GitHub
+    repository. The new deploy key will be linked to the personal access token used
+    to authenticate. Removing the personal access token will remove the deploy key.
+
 Run the bootstrap for a repository on your personal GitHub account:
```

Comment:
Please move after the first bootstrap command

Reply:
done